code/modules/antagonists/heretic/magic/space_crawl.dm

```diff
@@ -60,10 +60,10 @@
  */
 /datum/action/cooldown/spell/jaunt/space_crawl/proc/try_enter_jaunt(turf/our_turf, mob/living/jaunter)
 	// Begin the jaunt
-	jaunter.notransform = TRUE
+	ADD_TRAIT(jaunter, TRAIT_NO_TRANSFORM, REF(src))
 	var/obj/effect/dummy/phased_mob/holder = enter_jaunt(jaunter, our_turf)
-	if(!holder)
-		jaunter.notransform = FALSE
+	if(isnull(holder))
```

Comment:
```suggestion
	if(!holder)
```
Gonna sound silly but I'm fairly certain a few places (erroneously) return `FALSE` instead of `null` with this, so you should either leave it or go over and make sure everything has explicit `null` returns

Reply:
i looked at jaunt code and the only return there is intended to be instantiated. i can look at it again but i did check

Reply:
https://github.com/tgstation/tgstation/blob/671ac5b0d314de1233552a59acf7fa3ac120a9fb/code/modules/antagonists/heretic/magic/mirror_walk.dm#L78-L82

This was the one that sketched me out, different heretic spell turns out.

Reply:
yeah it's nerf or nothing here
![image](https://github.com/tgstation/tgstation/assets/34697715/14585848-5789-48aa-b910-95a0697aa153)

Reply:
yeah for some reason jaunt code is copypasted in two different spots, noticed that while doing code for this. should tackle that at a later date